integrationtests/Paket.IntegrationTests/PackSpecs.fs

```diff
@@ -349,3 +349,19 @@ let ``#1596 pack works for reflected definition assemblies``() =
     
     File.Exists(Path.Combine(outPath, "lib", "net45", "Project2.dll")) |> shouldEqual true
     File.Delete(templatePath)
+
+[<Test>]
+let ``#1806 pack localized dll`` () =
+    let scenario = "i001806-pack-localized"
+    let rootPath = scenarioTempPath scenario
+    let outPath = Path.Combine(rootPath, "out")
+    let package = Path.Combine(outPath, "LocalizedLib.1.0.0.0.nupkg")
+    
+    paket ("pack -v output \"" + outPath) scenario |> ignore
+    ZipFile.ExtractToDirectory(package, outPath)
+
+    Path.Combine(outPath, "lib", "net45", "LocalizedLib.dll") |> checkFileExists
```

Comment:
Could you use </> operator defined in Paket.IO (it is AutoOpen so you don't even have to open the namespace):

outPath </> "lib" </> "net45" </> "LocalizedLib.dll" |> checkFileExists